pkg/register/restResolver.go

```diff
@@ -69,7 +70,7 @@ func GetResolverFromEnv() (string, error) {
 }
 
 // GetDocument fetch a document from the resolver by DID identifier
-func (c *RestResolverClient) GetDocument(documentID string) (*RegisterDocument, error) {
+func (c *RestResolverClient) GetDocument(ctx context.Context, documentID string) (*RegisterDocument, error) {
```

Comment:
`NewRequestWithContext` + `Client.Do` (for Get & Register)

Reply:
[5ed827d](https://github.com/Iotic-Labs/iotics-identity-go/pull/46/commits/5ed827de576eabba85dc509573dcd950770cb73e)

Reply:
(see updated comment - a little more to do)